uint8 object_id
---
bool success